app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/PagerPageHolder.kt

```diff
@@ -416,7 +429,12 @@ class PagerPageHolder(
 
         imageStream.close()
         imageStream2.close()
-        return ImageUtil.mergeBitmaps(imageBitmap, imageBitmap2, isLTR, viewer.config.pageCanvasColor) {
+
+        val centerMargin = if (viewer.config.centerMarginType and PagerConfig.CenterMarginType.DOUBLE_PAGE_CENTER_MARGIN > 0 &&
+            !viewer.config.imageCropBorders
+        ) 96 / (Math.max(1, getHeight()) / Math.max(height, height2)) else 0
```

Comment:
No need to use the Math class, there are top level functions for this.
```suggestion
        ) 96 / (max(1, getHeight()) / max(height, height2)) else 0
```